#this message will be used for motor data information
MotorData[] motorData